packages/adapters/examples/segmentationStack/index.ts

```diff
@@ -1,226 +1,233 @@
+  const instructions = document.createElement('p');
+  instructions.innerText = `- ${text}`;
+  info.appendChild(instructions);
 }
 
 const state = {
-    renderingEngine: null,
-    renderingEngineId: "MY_RENDERING_ENGINE_ID",
-    toolGroup: null,
-    toolGroupId: "MY_TOOL_GROUP_ID",
-    viewportIds: ["CT_AXIAL"],
-    segmentationId: "LOAD_SEG_ID:" + cornerstone.utilities.uuidv4(),
-    referenceImageIds: [],
-    segImageIds: [],
-    skipOverlapping: false,
-    devConfig: { ...dicomMap.values().next().value }
+  renderingEngine: null,
+  renderingEngineId: 'MY_RENDERING_ENGINE_ID',
+  toolGroup: null,
+  toolGroupId: 'MY_TOOL_GROUP_ID',
+  viewportIds: ['CT_AXIAL'],
+  segmentationId: 'LOAD_SEG_ID:' + cornerstone.utilities.uuidv4(),
+  referenceImageIds: [],
+  segImageIds: [],
+  skipOverlapping: false,
+  devConfig: { ...dicomMap.values().next().value },
 };
 
-viewportGrid.addEventListener("dragover", evt => handleDragOver(evt), false);
+viewportGrid.addEventListener('dragover', (evt) => handleDragOver(evt), false);
 viewportGrid.addEventListener(
-    "drop",
-    evt => handleFileSelect(evt, state),
-    false
+  'drop',
+  (evt) => handleFileSelect(evt, state),
+  false
 );
 
 function loadDicom() {
-    restart(state);
+  restart(state);
 
-    const { toolGroup, viewportIds, renderingEngineId, renderingEngine } =
-        state;
+  const { toolGroup, viewportIds, renderingEngineId, renderingEngine } = state;
 
-    toolGroup.addViewport(viewportIds[0], renderingEngineId);
+  toolGroup.addViewport(viewportIds[0], renderingEngineId);
 
-    const viewport = state.renderingEngine.getStackViewport(viewportIds[0]);
+  const viewport = state.renderingEngine.getStackViewport(viewportIds[0]);
 
-    viewport.setStack(state.referenceImageIds);
-    cornerstoneTools.utilities.stackContextPrefetch.enable(element1);
+  viewport.setStack(state.referenceImageIds);
+  cornerstoneTools.utilities.stackContextPrefetch.enable(element1);
 
-    renderingEngine.render();
+  renderingEngine.render();
 }
 
 function createSegmentationRepresentation() {
-    csToolsSegmentation.addLabelmapRepresentationToViewport(
-        state.viewportIds[0],
-        [{ segmentationId: state.segmentationId }]
-    );
+  csToolsSegmentation.addLabelmapRepresentationToViewport(
+    state.viewportIds[0],
+    [{ segmentationId: state.segmentationId }]
+  );
 }
 
 // ============================= //
 addButtonToToolbar({
-    id: "LOAD_DICOM",
-    title: "Load DICOM",
-    onClick: async () => {
-        state.referenceImageIds = await createImageIdsAndCacheMetaData(
-            state.devConfig.fetchDicom
-        );
-
-        loadDicom();
-    },
-    container: group1
+  id: 'LOAD_DICOM',
+  title: 'Load DICOM',
+  onClick: async () => {
+    state.referenceImageIds = await createImageIdsAndCacheMetaData(
+      state.devConfig.fetchDicom
+    );
+
+    loadDicom();
+  },
+  container: group1,
 });
 
+let segmentationLoaded = false;
```

Comment:
This remembers whether the segmentation has been loaded, and prevents the stack exception reported in the issue - this was a pre-existing issue that occurs on a double click on that button.  No other changes.